src/lib/Libwin/popen.c

```diff
@@ -177,7 +184,9 @@ win_popen(char	*cmd, const char *type, struct pio_handles *pio, struct proc_ctrl
 		else
 			fsuccess = CreateProcess(NULL, cmd_line, &sa, &sa, proc_info->bInheritHandle,
 				CREATE_NO_WINDOW, NULL, NULL, &si, &pio->pi);
-
+		if (fsuccess == 0) {
+			log_err(-1, __func__, "failed in CreateProcess");	
```

Comment:
this condition doesn't seem right.
did you mean if(!fsuccess)(...

Reply:
No this is correct, as per https://docs.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-createprocessa